src/hooks-log.coffee

```diff
@@ -0,0 +1,23 @@
+util = require 'util'
+
+hooksLog = (logs = [], logger, logVariant, content) ->
+  if arguments.length is 4 and logVariant in ['info', 'debug', 'warn', 'verbose', 'error', 'log']
```

Comment:
Ok, I get it, but where is this functionality used? Is it exposed to hooks users? I think that user facing `hook.log` should preserve same interface as native `console.log`.